src/interactive/repl.ts

```diff
@@ -65,7 +64,19 @@ async function stopREPL(onDeactivate=false) {
             const sessionName = parseSessionArgs(config.get('persistentSession.tmuxSessionName'))
             const killSession = await confirmKill()
             if (killSession) {
-                await exec(`tmux kill-session -t ${sessionName}`)
+                const shellPath: string= config.get('persistentSession.shell')
+                const shellArgs: string[] = [<string>config.get('persistentSession.shellExecutionArgument'),`sessionNameSanitized=\`echo "${sessionName}" | sed "s/\\./_/g" \` && tmux kill-session -t $sessionNameSanitized && ${shellPath}`]
+                const env: any = {
+                    JULIA_EDITOR: getEditor()
+                }
+                vscode.window.createTerminal({
+                    name: `Julia REPL killed`,
+                    shellPath:  shellPath,
+                    shellArgs: shellArgs,
+                    isTransient: true,
+                    env: env,
+                    hideFromUser: true,
+                } as any)
```

Comment:
Can you explain why this is necessary? Invoking `tmux` directly should work fine, no?

Reply:
Invoking tmux with `exec` would see an empty `workspaceFolderBasename`.